packages/app/app/components/SearchResults/TracksResults/index.js

```diff
@@ -21,7 +21,7 @@ const TracksResults = ({ tracks, limit }) => {
             {_.take(collection, limit).map((track, index) => {
               if (
                 track &&
-                _.hasIn(track, 'name') & _.hasIn(track, 'image') &&
+                _.hasIn(track, 'name') && _.hasIn(track, 'image') &&
```

Comment:
Has this even worked before?

Reply:
Yes it's worked well,  `_.hasIn(track, 'name') & _.hasIn(track, 'image')` return only `1` or `0` cos `hasIn` return boolean and only case for it `return 1` is both give `true`.  As a whole expression only return true, 0 or false which perfectly works with this `if`. 

This change just make the expression to be consistent return `false` instead of `0`.